### PR TITLE
Update to version 2.3.0, update recipe

### PR DESCRIPTION
**Pull request by dbast**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,20 +34,35 @@ test:
...
+  summary: Creates classes with reference semantics, similar to R's built-in reference classes.
+    Compared to reference classes, R6 classes are simpler and lighter-weight, and they
+    are not built on S4 classes so they do not require the methods package. These classes
+    allow public and private members, and they support inheritance, even when the classes
+    are defined in different packages.
   license_family: MIT
 
 extra:
   recipe-maintainers:
+    - conda-forge/r
```
Comment (ocefpaf) [inline recipe/meta.yaml]: You can remove me and leave only the `r` team n PRs like that.
I do have access to all feedstocks anyway ;-p

Comment (dbast) [reply recipe/meta.yaml]: Ok, fine. Done :)

Comment (ocefpaf) [reply recipe/meta.yaml]: Thanks!

**Comment by dbast** [comment]
@ocefpaf ready to merge :)

**Comment by bgruening** [comment]
@ocefpaf is always super busy :) I will take care of this :)

**Comment by ocefpaf** [comment]
> @ocefpaf is always super busy :) I will take care of this :)

Not super busy at the moment, just flaky connection to the internet. (On "vacation" until next week.)

**Pull request merged**